.github/workflows/continuous-integration.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           coverageCommand: xvfb-run -a yarn run cover
           coverageLocations: ${{github.workspace}}/coverage/lcov.info:lcov
 
-      - run: yarn build-storybook
+      - run: yarn build-test-storybook
```

Comment:
[F] The visual components with any `act` behaviour failed to render on Chromatic because `act(…) is not supported in production version of React`. I have added another addon and set the NODE_ENV to test before building the storybook.

Reply:
Why would the storybook build be trying to compile something with `act`? Are we importing test modules in our stories?

Reply:
Just looked into it and seems it's only the use of `fireEvent` in stories that's the problem, they render correctly without this script if you replace `fireEvent.click(el)` with `el.click()` and we don't have to deploy a big ol' test build for every chromatic. 
I'll leave whatever solution you prefer up to you, but using `testing-library` utils for the fake dom in the real dom of storybook does feel weird to me.

Reply:
I should have left a better helper comment here.

I will try that. We need to be aware that not having the events wrapped in `act` could make the test flaky.

Reply:
That might not actually be true because we use the delay option to control when we take the screenshot.